h/views/api_auth.py

```diff
@@ -49,26 +87,15 @@ def get(self):
 
         state = credentials.get('state')
         user = self.user_svc.fetch(self.request.authenticated_userid)
+        response_mode = credentials.get('request').response_mode
```

Comment:
What’s `request` at this point? This is the `oauthlib` request, right, rather than anything to do with the HTTP request?